src/curl.cpp

```diff
@@ -113,6 +102,26 @@ namespace powerloader
         }
     }
 
+    CURLHandle::CURLHandle(CURLHandle&& rhs)
```

Comment:
I think that both move functions could be defaulted ` = default` if the error buffer was a vector and some way to wrap the pointers to set them at null on move would be set.
Not necesarilly an improvement to do now, but that might help in the future to not have to maintain these functions.